apps/wallet-dashboard/app/globals.css

```diff
@@ -110,6 +110,7 @@ body {
 
     @screen md {
         .home-page-grid-container {
+            height: calc(100vh - 140px);
```

Comment:
i would still make a min height so this doesnt happen 
![image](https://github.com/user-attachments/assets/77d8f2fd-1a49-4a39-8ad8-e0bbe077f1f0)